code/processes/metrics.q

```diff
@@ -0,0 +1,102 @@
+/ schemas for tables
```

Comment:
Why is this file in here? This is FSP isn't it?

Reply:
Can you address this one please? Is this supposed to be here?

Reply:
It's gone now